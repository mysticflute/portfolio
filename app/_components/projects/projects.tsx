import { type ProjectMetadata } from '@/lib/projects';
import Project from '@/components/project/project';
import styles from './projects.module.css';

type Props = {
  /**
   * The metadata about all portfolio projects.
   */
  projects: ProjectMetadata[];
};

export default function Projects({ projects }: Props) {
  return (
    <section className={styles.container}>
      <div className={styles.heading}>
        {/*prettier-ignore */}
        <h2 id="portfolio" className="textHeadingMedium">
          Take a look at my <span className="textHighlight3">music portfolio</span>
        </h2>
      </div>
      <div className={styles.list}>
        {projects.map(project => (
          <Project project={project} key={project.slug}></Project>
        ))}
      </div>
    </section>
  );
}
