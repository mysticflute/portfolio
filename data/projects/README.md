# Project Listing

This directory contains the data for music projects.

## Adding New Projects

1. Copy the `_template.yaml` file and rename it.
2. Edit the copied yaml file with the relevant information.
3. To find SoundCloud track IDs:

   1. Load the track page on soundcloud.com.
   2. Click the "Share" button.
   3. Click the "Embed" tab.
   4. Copy the code and paste it in a text editor.
   5. Copy the ten digit code that comes after the `/tracks` part of the src URL.<br/>
      For example, ...api.soundcloud.com/tracks/**1272228361**&color...

4. Add project images to the `public/images/project` directory.
5. If applicable, add new client logo images to the `public/images/logo` directory.

## Disabling a project

1. Rename the yaml file to start with an underscore, e.g., `_project.yaml`.
