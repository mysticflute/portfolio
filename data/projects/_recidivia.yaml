# Name of the project.
name: Recidivia

# My primary role on the project.
# Examples: Composer, Artist, Producer, Arranger.
role: Composer

# Short project description.
description: >
  In ultricies viverra sed at hendrerit drogon nunc scelerisque nisl
  pellentesque et dignissim et aenean tempor adipiscing eget mi diam at tempus.

# The main image for the project.
# Must be a file under public/images/projects.
# If an image is not available yet, use "placeholder.svg".

image: stencil-computer.svg

# A link to an external project landing page (optional).
link: https://twitter.com/RecidiviaRPG

# The client's logo image (optional).
# Must be a file under public/images/logos.
logo: twitch.svg

### NOT CURRENTLY IMPLEMENTED ###

# Project type (Video Game, Visual Novel, Film, Podcast, Album, etc).
# type: Video Game

# URL path for the project detail page.
# slug: project-name

# Large image for the project detail page.
# largeImage: placeholder-large.png

# Long project description for the detail page.
# longDescription: >
#   In ultricies viverra sed at hendrerit drogon nunc scelerisque nisl
#   pellentesque et dignissim et aenean tempor adipiscing eget mi diam at tempus.

# Project's accent color (optional).
# Must be one of: yellow, blue, purple, red, green, white, black, grey.
# color: purple

# My additional roles on the project (optional).
# additionalRoles:
#   - Composer
#   - Producer
#   - Arranger

# Platforms the project is released on (optional).
# platforms:
#   - Steam
#   - Nintendo Switch
#   - Playstation 5

# just here because of this bug:
# https://github.com/prettier/prettier/issues/10922
dummy: value
