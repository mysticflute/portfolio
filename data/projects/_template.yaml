# Name of the project.
name: Project Name

# Unique, dash-delimited name for the project URL (optional).
# If none specified, defaults to the filename.
# slug: project-name

# The sort position for the project.
# sort: 1

### NOT CURRENTLY IMPLEMENTED ###
# The project type.
# Examples: Video Game, Visual Novel, Film, TV, Documentary,
# Podcast, Streamer, Board Game, Album, Compilation.
# type: Video Game

# My primary role on the project.
# Examples: Composer, Primary Composer, Additional Composer,
# Artist, Producer, Arranger, Sound Designer, Music Implementor,
# Scoring Assistant, Composer's Assistant, Creator.
role: Composer

### NOT CURRENTLY IMPLEMENTED ###
# My additional roles on the project (optional).
# additionalRoles:
#   - Composer
#   - Producer
#   - Arranger
#   - Sound Designer

# Short project description.
description: >
  In ultricies viverra sed at hendrerit drogon nunc scelerisque nisl
  pellentesque et dignissim et aenean tempor adipiscing eget mi diam at tempus.

# List of 10-digit SoundCloud track IDs. (optional).
# See the README.md file for how to find these.
# soundCloudIds:
#   - 1272228361
#   - 1269466039

# An icon that represents the game type (optional).
# Must be a file under public/icons.
# icon: pulsar/game-controller.svg

### NOT CURRENTLY IMPLEMENTED ###
# The main image for the project (optional).
# Must be a file under public/images/projects.
# image: placeholder.svg

### NOT CURRENTLY IMPLEMENTED ###
# A large image for the project detail page (optional).
# Must be a file under public/images/projects.
# largeImage: placeholder-large.png

### NOT CURRENTLY IMPLEMENTED ###
# A link to an external project landing page (optional).
# link: https://twitter.com/SquareEnix

### NOT CURRENTLY IMPLEMENTED ###
# The date the project was or will be completed (optional).
# projectDate: 1/1/2025

### NOT CURRENTLY IMPLEMENTED ###
# The date the project was or will be released (optional).
# releaseDate: 1/1/2025

### NOT CURRENTLY IMPLEMENTED ###
# Platforms the project is released on (optional).
# platforms:
#   - Steam
#   - Nintendo Switch
#   - Playstation 5

# Project's accent color, in css hex format (optional).
# color: '#5c42fb'

### NOT CURRENTLY IMPLEMENTED ###
# Long project description for the detail page.
# longDescription: >
#   In ultricies viverra sed at hendrerit drogon nunc scelerisque nisl
#   pellentesque et dignissim et aenean tempor adipiscing eget mi diam at tempus.

# just here because of this bug:
# https://github.com/prettier/prettier/issues/10922
dummy: value
