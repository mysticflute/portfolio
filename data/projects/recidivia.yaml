# Name of the project.
name: Recidivia

# Unique, dash-delimited name for the project URL (optional).
# If note specified, defaults to the filename.
# slug: project-name

# The sort position for the project.
sort: 1

### NOT CURRENTLY IMPLEMENTED ###
# The project type.
# Examples: Video Game, Visual Novel, Film, TV, Documentary,
# Podcast, Streamer, Board Game, Album, Compilation.
# type: Video Game

# My primary role on the project.
# Examples: Composer, Primary Composer, Additional Composer,
# Artist, Producer, Arranger, Sound Designer, Music Implementor,
# Scoring Assistant, Composer's Assistant, Creator.
role: Primary Composer

### NOT CURRENTLY IMPLEMENTED ###
# My additional roles on the project (optional).
# additionalRoles:
#   - Composer
#   - Producer
#   - Arranger

# Short project description.
description: >
  An old-school 2D RPG adventure! Developed by 1ManArmE Studios, this is the
  story of a Temple Guard's desperate attempt to repent for actions he believes
  to be no fault of his own.

# List of 10-digit SoundCloud track IDs. (optional).
# See the README.md file for how to find these.
soundcloudIDs:
  - 1468989199
  - 1468987642
  - 1468969438
  - 1468955629
  - 1468955386

# The main image for the project.
# Must be a file under public/images/projects.
# If an image is not available yet, use placeholder.svg.
image: recidivia-onyx.jpg

### NOT CURRENTLY IMPLEMENTED ###
# Large image for the project detail page.
# largeImage: placeholder-large.png

# The client's logo image (optional).
# Must be a file under public/images/logos.
logoImage: logo-twitch.svg

# A link to an external project landing page (optional).
link: https://twitter.com/RecidiviaRPG

### NOT CURRENTLY IMPLEMENTED ###
# The date the project was or will be released (optional).
# releaseDate: 1/1/2025

### NOT CURRENTLY IMPLEMENTED ###
# Platforms the project is released on (optional).
# platforms:
#   - Steam
#   - Nintendo Switch
#   - Playstation 5

# Project's accent color, in css hex format (optional).
color: '#5c42fb'
# color: '#920025'

### NOT CURRENTLY IMPLEMENTED ###
# Long project description for the detail page.
# longDescription: >
#   In ultricies viverra sed at hendrerit drogon nunc scelerisque nisl
#   pellentesque et dignissim et aenean tempor adipiscing eget mi diam at tempus.

# just here because of this bug:
# https://github.com/prettier/prettier/issues/10922
dummy: value
