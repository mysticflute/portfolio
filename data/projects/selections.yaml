# Name of the project or collection.
name: Selected Compositions

# Unique, dash-delimited name for the project URL (optional).
# If none specified, defaults to the filename.
slug: selected-compositions

# The sort position for the project.
sort: 1

### NOT CURRENTLY IMPLEMENTED ###
# The project type.
# Examples: Video Game, Visual Novel, Film, TV, Documentary,
# Podcast, Streamer, Board Game, Album, Compilation.
type: Compilation

# My primary role on the project.
# Examples: Composer, Primary Composer, Additional Composer,
# Artist, Producer, Arranger, Sound Designer, Music Implementor,
# Scoring Assistant, Composer's Assistant, Creator.
role: Composer

### NOT CURRENTLY IMPLEMENTED ###
# My additional roles on the project (optional).
# additionalRoles:
#   - Composer
#   - Producer
#   - Arranger
#   - Sound Designer

# Short project description.
description: >
  A selection of my compositions across various recent projects. Styles include
  instrumental, orchestral, atmospheric and more.

# List of audio tracks for this project or compilation.
# At least one embeddable url (mp3/aac) must be specified for each track.
tracks:
  # Display name for the track.
  - name: "Blue River Road"
    # The urls to embed, externally stream or buy the track.
    mp3: https://feeds.soundcloud.com/stream/1272228361

  # Display name for the track.
  - name: "No Escape (Vulgari OST)"
    # The urls to embed, externally stream or buy the track.
    mp3: https://feeds.soundcloud.com/stream/1363178761

  # Display name for the track.
  - name: "Overworld (Vulgari OST)"
    # The urls to embed, externally stream or buy the track.
    mp3: https://feeds.soundcloud.com/stream/1362994615

  # Display name for the track.
  - name: "Princess Aiysha"
    # The urls to embed, externally stream or buy the track.
    mp3: https://feeds.soundcloud.com/stream/1269487699

  # Display name for the track.
  - name: "Deep Dark Forest (Recidivia OST)"
    # The urls to embed, externally stream or buy the track.
    mp3: https://feeds.soundcloud.com/stream/1468989199

  # Display name for the track.
  - name: "Fireflies (Turning OST)"
    # The urls to embed, externally stream or buy the track.
    mp3: https://feeds.soundcloud.com/stream/1661983896

  # Display name for the track.
  - name: "The Secret (Vulgari OST)"
    # The urls to embed, externally stream or buy the track.
    mp3: https://feeds.soundcloud.com/stream/1363203541

  # Display name for the track.
  - name: "Sarvarā's Gate  (Vulgari OST)"
    # The urls to embed, externally stream or buy the track.
    mp3: https://feeds.soundcloud.com/stream/1363214545

  # Display name for the track.
  - name: "Chasing Freedom (Vulgari OST)"
    # The urls to embed, externally stream or buy the track.
    mp3: https://feeds.soundcloud.com/stream/1363207933

  # Display name for the track.
  - name: "Overworld Map (D&D Campaign)"
    # The urls to embed, externally stream or buy the track.
    mp3: https://feeds.soundcloud.com/stream/1510431769

# Project's accent color, in css hex format (optional).
color: "#ffab91"

# An icon that represents the game type (optional).
# Must be a file under public/icons.
icon: pulsar/favorites.svg
# icon: pulsar/bookmark.svg

### NOT CURRENTLY IMPLEMENTED ###
# The main image for the project (optional).
# Must be a file under public/images/projects.
# image: placeholder.svg

### NOT CURRENTLY IMPLEMENTED ###
# A large image for the project detail page (optional).
# Must be a file under public/images/projects.
# largeImage: placeholder-large.png

# A link to an external project landing page (optional).
# link: https://twitter.com/SquareEnix

### NOT CURRENTLY IMPLEMENTED ###
# The date the project was or will be completed (optional).
# projectDate: 1/1/2025

### NOT CURRENTLY IMPLEMENTED ###
# The date the project was or will be released (optional).
# releaseDate: 1/1/2025

### NOT CURRENTLY IMPLEMENTED ###
# Platforms the project is released on (optional).
# platforms:
#   - Steam
#   - Nintendo Switch
#   - Playstation 5

### NOT CURRENTLY IMPLEMENTED ###
# Long project description for the detail page.
# longDescription: >
#   In ultricies viverra sed at hendrerit drogon nunc scelerisque nisl
#   pellentesque et dignissim et aenean tempor adipiscing eget mi diam at tempus.

# just here because of this bug:
# https://github.com/prettier/prettier/issues/10922
dummy: value
