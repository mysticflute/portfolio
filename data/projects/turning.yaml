# Name of the project or collection.
name: Turning

# Unique, dash-delimited name for the project URL (optional).
# If none specified, defaults to the filename.
# slug: project-name

# The sort position for the project.
sort: 98

### NOT CURRENTLY IMPLEMENTED ###
# The project type.
# Examples: Video Game, Visual Novel, Film, TV, Documentary,
# Podcast, Streamer, Board Game, Album, Compilation.
type: Video Game

# My primary role on the project.
# Examples: Composer, Primary Composer, Additional Composer,
# Artist, Producer, Arranger, Sound Designer, Music Implementor,
# Scoring Assistant, Composer's Assistant, Creator.
role: Additional Composer

### NOT CURRENTLY IMPLEMENTED ###
# My additional roles on the project (optional).
# additionalRoles:
#   - Composer
#   - Producer
#   - Arranger
#   - Sound Designer

# Short project description.
description: >
  A combat-less game about agriculture, economy, and exploration developed for
  fans of Stardew Valley, Forager, and A Link to the Past.

# List of audio tracks for this project or compilation.
# At least one embeddable url (mp3/aac) must be specified for each track.
tracks:
  # Display name for the track.
  - name: "Seeds of Life [Morning]"
    # The urls to embed, externally stream or buy the track.
    aac: https://res.cloudinary.com/deiecdwbk/video/private/br_192k:constant/v1728595851/nxlkskdf74gzjgxy32h7.aac
    mp3: https://res.cloudinary.com/deiecdwbk/video/private/br_192k:constant/v1728595851/nxlkskdf74gzjgxy32h7.mp3

  # Display name for the track.
  - name: "Sunny Shovel [Morning 2]"
    # The urls to embed, externally stream or buy the track.
    aac: https://res.cloudinary.com/deiecdwbk/video/private/br_192k:constant/v1728596042/fsd7rwnyadfvicwefgmt.aac
    mp3: https://res.cloudinary.com/deiecdwbk/video/private/br_192k:constant/v1728596042/fsd7rwnyadfvicwefgmt.mp3

  # Display name for the track.
  - name: "After the Clouds [Afternoon]"
    # The urls to embed, externally stream or buy the track.
    aac: https://res.cloudinary.com/deiecdwbk/video/private/br_192k:constant/v1728595779/vziix3j6vecxgup1fcja.aac
    mp3: https://res.cloudinary.com/deiecdwbk/video/private/br_192k:constant/v1728595779/vziix3j6vecxgup1fcja.mp3

  # Display name for the track.
  - name: "The Frog's Dream [Evening]"
    # The urls to embed, externally stream or buy the track.
    aac: https://res.cloudinary.com/deiecdwbk/video/private/br_192k:constant/v1728596043/x8e0evys2gthf0dw8irh.aac
    mp3: https://res.cloudinary.com/deiecdwbk/video/private/br_192k:constant/v1728596043/x8e0evys2gthf0dw8irh.mp3

  # Display name for the track.
  - name: "Fireflies [Night]"
    # The urls to embed, externally stream or buy the track.
    aac: https://res.cloudinary.com/deiecdwbk/video/private/br_192k:constant/v1728595781/vbjwnmuuswd03a3fhvks.aac
    mp3: https://res.cloudinary.com/deiecdwbk/video/private/br_192k:constant/v1728595781/vbjwnmuuswd03a3fhvks.mp3

  # Display name for the track.
  - name: "A Day's Work"
    # The urls to embed, externally stream or buy the track.
    aac: https://res.cloudinary.com/deiecdwbk/video/private/br_192k:constant/v1728595665/x27qavxemlquxjvokeyy.aac
    mp3: https://res.cloudinary.com/deiecdwbk/video/private/br_192k:constant/v1728595665/x27qavxemlquxjvokeyy.mp3

  # Display name for the track.
  - name: "Forest"
    # The urls to embed, externally stream or buy the track.
    aac: https://res.cloudinary.com/deiecdwbk/video/private/br_192k:constant/v1728595781/weju4sros9io3qcxg8fm.aac
    mp3: https://res.cloudinary.com/deiecdwbk/video/private/br_192k:constant/v1728595781/weju4sros9io3qcxg8fm.mp3

  # Display name for the track.
  - name: "Silent Awakening"
    # The urls to embed, externally stream or buy the track.
    aac: https://res.cloudinary.com/deiecdwbk/video/private/br_192k:constant/v1728595849/vne7g0q6thjvmwxsw3ow.aac
    mp3: https://res.cloudinary.com/deiecdwbk/video/private/br_192k:constant/v1728595849/vne7g0q6thjvmwxsw3ow.mp3

  # Display name for the track.
  - name: "Passage"
    # The urls to embed, externally stream or buy the track.
    aac: https://res.cloudinary.com/deiecdwbk/video/private/br_192k:constant/v1728595847/qdp5hxluvcsax54pomts.aac
    mp3: https://res.cloudinary.com/deiecdwbk/video/private/br_192k:constant/v1728595847/qdp5hxluvcsax54pomts.mp3

  # Display name for the track.
  - name: "Determination"
    # The urls to embed, externally stream or buy the track.
    aac: https://res.cloudinary.com/deiecdwbk/video/private/br_192k:constant/v1728595778/crushrlnax33s4b0wyee.aac
    mp3: https://res.cloudinary.com/deiecdwbk/video/private/br_192k:constant/v1728595778/crushrlnax33s4b0wyee.mp3

# Project's accent color, in css hex format (optional).
color: "#f49ac1"

# An icon that represents the game type (optional).
# Must be a file under public/icons.
icon: pulsar/leaf.svg

### NOT CURRENTLY IMPLEMENTED ###
# The main image for the project (optional).
# Must be a file under public/images/projects.
# image: placeholder.svg

### NOT CURRENTLY IMPLEMENTED ###
# A large image for the project detail page (optional).
# Must be a file under public/images/projects.
# largeImage: placeholder-large.png

### NOT CURRENTLY IMPLEMENTED ###
# A link to an external project landing page (optional).
link: https://www.turninggame.com

### NOT CURRENTLY IMPLEMENTED ###
# The date the project was or will be completed (optional).
# projectDate: 1/1/2025

### NOT CURRENTLY IMPLEMENTED ###
# The date the project was or will be released (optional).
# releaseDate: 1/1/2025

### NOT CURRENTLY IMPLEMENTED ###
# Platforms the project is released on (optional).
# platforms:
#   - Steam
#   - Nintendo Switch
#   - Playstation 5

### NOT CURRENTLY IMPLEMENTED ###
# Long project description for the detail page.
# longDescription: >
#   In ultricies viverra sed at hendrerit drogon nunc scelerisque nisl
#   pellentesque et dignissim et aenean tempor adipiscing eget mi diam at tempus.

# just here because of this bug:
# https://github.com/prettier/prettier/issues/10922
dummy: value
