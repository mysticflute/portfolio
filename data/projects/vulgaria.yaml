# Name of the project.
name: Vulgaria

# Unique, dash-delimited name for the project URL (optional).
# If note specified, defaults to the filename.
# slug: project-name

# The sort position for the project.
sort: 2

### NOT CURRENTLY IMPLEMENTED ###
# The project type.
# Examples: Video Game, Visual Novel, Film, TV, Documentary,
# Podcast, Streamer, Board Game, Album, Compilation.
# type: Video Game

# My primary role on the project.
# Examples: Composer, Primary Composer, Additional Composer,
# Artist, Producer, Arranger, Sound Designer, Music Implementor,
# Scoring Assistant, Composer's Assistant, Creator.
role: Composer

### NOT CURRENTLY IMPLEMENTED ###
# My additional roles on the project (optional).
# additionalRoles:
#   - Composer
#   - Producer
#   - Arranger

# Short project description.
description: >
  A top-down adventure game, developed by Statson and releasing in 2023.
  Features strings, flutes, harps, taikos and hand perc.

# List of 10-digit SoundCloud track IDs. (optional).
# See the README.md file for how to find these.
soundcloudIDs:
  - 1363207933
  - 1362994615
  - 1363175623
  - 1363178761
  - 1363186105
  - 1363189672
  - 1363214545
  - 1363203541

# The main image for the project.
# Must be a file under public/images/projects.
# If an image is not available yet, use placeholder.svg.
image: stencil-computer.svg

### NOT CURRENTLY IMPLEMENTED ###
# Large image for the project detail page.
# largeImage: placeholder-large.png

# The client's logo image (optional).
# Must be a file under public/images/logos.
logoImage: logo-twitch.svg

# A link to an external project landing page (optional).
link: https://twitter.com/StatsonVulgaira

### NOT CURRENTLY IMPLEMENTED ###
# The date the project was or will be released (optional).
# releaseDate: 1/1/2025

### NOT CURRENTLY IMPLEMENTED ###
# Platforms the project is released on (optional).
# platforms:
#   - Steam
#   - Nintendo Switch
#   - Playstation 5

# Project's accent color, in css hex format (optional).
color: '#5c42fb'

### NOT CURRENTLY IMPLEMENTED ###
# Long project description for the detail page.
# longDescription: >
#   In ultricies viverra sed at hendrerit drogon nunc scelerisque nisl
#   pellentesque et dignissim et aenean tempor adipiscing eget mi diam at tempus.

# just here because of this bug:
# https://github.com/prettier/prettier/issues/10922
dummy: value
