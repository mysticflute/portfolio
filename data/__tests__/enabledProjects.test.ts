import { describe, it, expect } from '@jest/globals';
import { getProjects } from '@/lib/projects';

describe('currently enabled projects', () => {
  it('should match snapshot', async () => {
    const projects = await getProjects();

    // generalize generated fields
    const propertyMatchers = projects.map(project => {
      if (project.tracks) {
        return {
          tracks: project.tracks.map(() => ({ id: expect.any(String) })),
        };
      }
      return {};
    });

    expect(projects).toMatchSnapshot(propertyMatchers);
  });
});
