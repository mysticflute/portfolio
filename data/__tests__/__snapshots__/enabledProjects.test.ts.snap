// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`currently enabled projects should match snapshot 1`] = `
[
  {
    "color": "#ffab91",
    "description": "A selection of my compositions across various recent projects. Styles include instrumental, orchestral, atmospheric and more.
",
    "icon": "pulsar/favorites.svg",
    "name": "Selected Compositions",
    "role": "Composer",
    "slug": "selected-compositions",
    "sort": 1,
    "tracks": [
      {
        "mp3": "https://feeds.soundcloud.com/stream/1272228361",
        "name": "Blue River Road",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1363178761",
        "name": "No Escape (Vulgari OST)",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1362994615",
        "name": "Overworld (Vulgari OST)",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1269487699",
        "name": "Princess Aiysha",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1468989199",
        "name": "Deep Dark Forest (Recidivia OST)",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1661983896",
        "name": "Fireflies (Turning OST)",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1363203541",
        "name": "The Secret (Vulgari OST)",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1363214545",
        "name": "Sarvarā's Gate [Final Stage] (Vulgari OST)",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1363207933",
        "name": "Chasing Freedom (Vulgari OST)",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1510431769",
        "name": "Overworld Map (JRPG-Style Dungeons and Dragons Campaign)",
      },
    ],
  },
  {
    "color": "#f49ac1",
    "description": "A combat-less game about agriculture, economy, and exploration developed for fans of Stardew Valley, Forager, and A Link to the Past.
",
    "icon": "pulsar/leaf.svg",
    "link": "https://www.turninggame.com",
    "name": "Turning",
    "role": "Additional Composer",
    "slug": "turning",
    "sort": 98,
    "tracks": [
      {
        "mp3": "https://feeds.soundcloud.com/stream/1661974287",
        "name": "Seeds of Life [Morning]",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1661979261",
        "name": "Sunny Shovel [Morning 2]",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1661980971",
        "name": "After the Clouds [Afternoon]",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1661982381",
        "name": "The Frog's Dream [Evening]",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1661983896",
        "name": "Fireflies [Night]",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1661985333",
        "name": "A Day's Work",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1661991063",
        "name": "Forest",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1661999643",
        "name": "Silent Awakening",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1661987727",
        "name": "Passage",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1661989830",
        "name": "Determination",
      },
    ],
  },
  {
    "color": "#5c42fb",
    "description": "A story-driven 2D RPG under development by 1ManArmE Studios. Contributions include multiple orchestral and instrumental tracks.
",
    "icon": "pulsar/crystal.svg",
    "link": "https://twitter.com/RecidiviaRPG",
    "name": "Recidivia",
    "role": "Primary Composer",
    "slug": "recidivia",
    "sort": 99,
    "tracks": [
      {
        "mp3": "https://feeds.soundcloud.com/stream/1468989199",
        "name": "Deep Dark Forest",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1468987642",
        "name": "Penca Plains",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1468969438",
        "name": "Growing Suspicions",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1468955629",
        "name": "Inn A",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1468955386",
        "name": "Inn B",
      },
    ],
  },
  {
    "color": "#4b914b",
    "description": "A top-down, rogue-like adventure game developed by Statson. Features strings, flutes, harps, taikos and hand perc.
",
    "icon": "pulsar/adventure.svg",
    "name": "Vulgari",
    "role": "Composer",
    "slug": "vulgaria",
    "sort": 100,
    "tracks": [
      {
        "mp3": "https://feeds.soundcloud.com/stream/1363207933",
        "name": "Chasing Freedom",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1362994615",
        "name": "Overworld",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1363175623",
        "name": "Dungeon",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1363178761",
        "name": "No Escape",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1363186105",
        "name": "Umbra",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1363189672",
        "name": "Windy Cave",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1363214545",
        "name": "Sarvarā's Gate [Final Stage]",
      },
      {
        "mp3": "https://feeds.soundcloud.com/stream/1363203541",
        "name": "The Secret",
      },
    ],
  },
]
`;
